monorepo/website/src/pages/docs/concepts/metadataformat.mdx

```diff
@@ -1,156 +1,29 @@
 ---
 layout: ../../../layouts/DocLayout.astro
-title: "Metadata format"
+title: 'Metadata format'
 route: /
 ---
 
+import MetadataTable from '../../../components/MetadataTable.astro';
+
 Here you can find a brief description and example of each field you can include in the metadata file uploaded with your samples.
 Most of these fields are currently optional, but we do have some required fields.
 
-- [Required Fields](#required-fields)
-- [Optional Fields](#optional-fields)
-  - [Desired Fields](#desired-fields)
-  - [Sample Related Fields](#sample-related-fields)
-  - [Host Related Fields](#host-related-fields)
-  - [Sequencing Related Fields](#sequencing-related-fields)
-  - [Other Optional Fields](#other-optional-fields)
+## Metadata Types
 
 Some fields, like dates, countries and authors, will be standardized so that all entries in the database are easy to process.
 At the moment, these are the only fields we standardize, and we will keep users updated if that changes in the future.
 
+### Dates
+
 {/* Please format dates as YYYY-MM-DD - use XX if unknown, ex: `2020-03-XX` or `20XX-XX-XX`. */}
 Please format dates as YYYY-MM-DD.
 To submit sequences, the full date (including day) is required. If you have older sequences for which the full date isn't available, or legal reasons relating to releasing the full date, please get in touch via [submission@pathoplexus.org](mailto:submission@pathoplexus.org).
```

Comment:
Is this still true? And I guess if it isn't technically, do we want to keep saying this to encourage it, or is it just miseading if we actually allow incomplete dates? (Which I think we do currently?)